src/py42/clients/alerts.py

```diff
@@ -1,3 +1,11 @@
+class AlertState(object):
```

Comment:
We already have an `AlertState` enum in `py42.sdk.queries.alerts.filters.alert_filter`.... I believe this are the same thing too, so we should leverage this. Correct me if I am  wrong

Reply:
Yea they are one and the same, good that you brought it up, Filters only supports `resolved` and `open` choices, need to add the new ones there.
I don't see any harm in using them, only that they were specifically suited for filters.

Reply:
Yeah I think that is ok in this case.  We are not using it as a filter here but it is still the same constants.

Reply:
FYI: Adding import statement for filters raises warning while generating docs. Investigating the error it seems need to add some path statements to `conf.py` in `docs`. Working on it.

```
WARNING: autodoc: failed to import class 'alerts.AlertsClient' from module 'py42.clients'; the following exception was raised:
cannot import name 'AlertsClient'
```

Reply:
Changed.